src/library/trackset/crate/cratetablemodel.cpp

```diff
@@ -103,17 +102,7 @@ bool CrateTableModel::addTrack(const QModelIndex& index, QString location) {
 }
 
 TrackModel::CapabilitiesFlags CrateTableModel::getCapabilities() const {
-    CapabilitiesFlags caps =  TRACKMODELCAPS_NONE
-            | TRACKMODELCAPS_RECEIVEDROPS
-            | TRACKMODELCAPS_ADDTOPLAYLIST
-            | TRACKMODELCAPS_ADDTOCRATE
-            | TRACKMODELCAPS_ADDTOAUTODJ
-            | TRACKMODELCAPS_EDITMETADATA
-            | TRACKMODELCAPS_LOADTODECK
-            | TRACKMODELCAPS_LOADTOSAMPLER
-            | TRACKMODELCAPS_LOADTOPREVIEWDECK
-            | TRACKMODELCAPS_REMOVE_CRATE
-            | TRACKMODELCAPS_RESETPLAYED;
+    CapabilitiesFlags caps = TRACKMODELCAPS_NONE | TRACKMODELCAPS_RECEIVEDROPS | TRACKMODELCAPS_ADDTOPLAYLIST | TRACKMODELCAPS_ADDTOCRATE | TRACKMODELCAPS_ADDTOAUTODJ | TRACKMODELCAPS_EDITMETADATA | TRACKMODELCAPS_LOADTODECK | TRACKMODELCAPS_LOADTOSAMPLER | TRACKMODELCAPS_LOADTOPREVIEWDECK | TRACKMODELCAPS_REMOVE_CRATE | TRACKMODELCAPS_RESETPLAYED;
```

Comment:
Unfortunately clang-format only doesn't recognize this variant. The operands are preserved on separate lines when placing the `|` operator *after* each operand.

This code has been written before the introduction of clang-format.

Reply:
Fixed :+1: